packages/bpk-docs/package-lock.json

```diff
@@ -1,11 +1,14 @@
 {
-	"requires": true,
+	"name": "bpk-docs",
```

Comment:
This `package-lock.json` has the package name in it again  😤 

I still do not know what causes this.

Reply:
😭 not again!

Reply:
Notice that my PR remvoes them -> https://github.com/Skyscanner/backpack/pull/256/files (cmd + f "package-lock.json", there are two examples)

Reply:
This page mentions, or at least implies, that the name will be part of the lockfile: https://docs.npmjs.com/files/package-locks (it's about halfway down the page).

Still no idea why it's happening just to me but at least we know it's a thing..